### PR TITLE
JW-123 port vr_packet.c

### PR DESCRIPTION
**Pull request by michal-clapinski**
Depends on codilime/contrail-sandesh#4

### CODE REVIEW

File: dp-core/vrouter.c [hunk 1]
```diff
@@ -352,19 +355,19 @@ vrouter_ops_get_process(void *s_req)
     /* Logging entries */
     resp->vo_log_level = vr_get_log_level();
     resp->vo_log_type_enable =
-        vr_get_enabled_log_types(&resp->vo_log_type_enable_size);
+        (int32_t*)vr_get_enabled_log_types((int32_t*)&resp->vo_log_type_enable_size);
```
Comment (sodar) [inline dp-core/vrouter.c]: Callback returns `unsigned int*` instead of `int *`

Comment (michal-clapinski) [inline dp-core/vrouter.c]: Unfortunately I don't understand what you mean. Could you elaborate? I don't see any warning in VS.

Comment (sodar) [inline dp-core/vrouter.c]: `vr_get_enabled_log_types` is defined as `vrouter_host->hos_get_enabled_log_types`. It is a callback function and it is defined as a function returning `unsigned int *` (https://github.com/codilime/contrail-vrouter/blob/dp-core-packet-c/include/vrouter.h#L206):

```C
// include/vrouter.h
struct host_os {
    // ....
    unsigned int *(*hos_get_enabled_log_types)(int *);
    // ...
};
```

I don't know why VS does not give any errors on this one.

Comment (sodar) [inline dp-core/vrouter.c]: I have checked `vr_get_enabled_log_types` implementation and it returns values based on binary masks, so signed/unsigned has no meaning here.

**Pull request merged**